platform/posix/man/yaml2ck (zoe's conflicted copy 2022-08-13).1

```diff
@@ -0,0 +1,58 @@
+.TH "yaml2ck" 1 "12 Aug 2022" "yaml2ck" \" -*- nroff -*-
```

Comment:
Not sure where this file came from. Once it is removed, this should be good to go!

Reply:
oops - looks like Dropbox strikes again